oak_functions/abi/src/lib.rs

```diff
@@ -27,33 +27,110 @@ use core::mem::size_of;
 use serde::{Deserialize, Serialize};
 
 pub mod proto {
```

Comment:
Is `proto::Response` still around? Do we need it? Or at the very least, does it need to be made public? (Do any of these need to be `pub`?)

Reply:
removed, thank you.